docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +42,100 @@ rg.init(workspace="my_shared_workspace")
 :::
 
-### Override default API key
+## Add new users and workspaces
+
+The above user management model is configured using the Argilla tasks, which server maintainers can define before launching an Argilla instance.
 
-To override the default API key you can set the following environment variable before launching the server:
+### Prepare the database
+First of all, you need to make sure that database tables and models are up-to-date. This task must be launched when a new version of Argilla is installed.
 
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_APIKEY=new-apikey
+python -m argilla.tasks.database.migrate
 ```
 
+:::{note}
+It is important to launch this task prior to any other database action.
+:::
 
-### Override default password
-
-To override the password, you must set an environment variable that contains an already hashed password.
-You can use `htpasswd` to generate a hashed password:
 
+### Creating an admin user
 ```bash
-htpasswd -nbB "" my-new-password
-:$apr1$n0C4S20a$noG.3yWxH1OIKfFITgzl30
+python -m argilla.tasks.users.create --role admin
 ```
 
-Afterwards, set the environment variable omitting the first `:` character (in our case `$apr1$n0C4...`):
-
+### Creating an annotator user
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_PASSWORD="<generated_user_password>"
+python -m argilla.tasks.users.create --role annotator
 ```
 
-Alternatively, you can also generate the hash using passlib's CryptContext: E.g., by running the following in a python console:
+### HOWTO Create a new workspace an assign it to an existing annotator
+
+#### 1. Setup a http client with an admin auth header
 
 ```python
-from passlib.context import CryptContext
-print(CryptContext(schemes=["bcrypt"], deprecated="auto").hash('password'))
+auth_headers = {"X-Argilla-API-Key": "argilla.apikey"}
+http = httpx.Client(base_url="http://localhost:6900", headers=auth_headers)
 ```
 
-## Add new users and workspaces
+#### 2. Call the create workspace endpoint
+
+```python
+workspace = http.post("/api/workspaces", json={"name": "new-workspace"}).json()
+workspace
+```
 
-To configure your Argilla instance for various users, you just need to create a yaml file as follows:
-```yaml
-#.users.yaml
-# Users are provided as a list
-- username: user1
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser1APIKEY"
-  workspaces: [] # This user will only have her user workspace available
-- username: user2
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser2APIKEY"
-  workspaces: ['client_projects'] # access to her user workspace and the client_projects workspace
-- username: user3
-  hashed_password: <generated-hashed-password> # See the previous section above
-  api_key: "ThisIsTheUser2APIKEY" # this user can access all workspaces
-- ...
+#### 3. Select which annotators will be linked to then new workspace and link them by using their ids
```

Comment:
Headers should be a bit shorter to make them readable in the docs and index them nicely Google. Perhaps. something like "Assign users to workspaces" would work.